requirements.txt

```diff
@@ -22,4 +22,4 @@ rioxarray==0.13.3
 s3fs==2022.11.0
 scipy>=1.2.0
 shapely==1.8.5
-xclim==0.41.0
+xclim==0.36.0
```

Comment:
We've been using `0.42.0` on the hub, so it would be good to go with that.

Reply:
Good to know-- thanks!